### PR TITLE
DOCS-8441 LLM Observability Terms and Concepts Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Updates the Evaluations section in the Terms and Concepts page to include the stage of evaluation, the method of evaluation, the meaning of the evaluation, and a screenshot of an example evaluation.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by davit-bar — APPROVED** [review]

File: content/en/llm_observability/terms/_index.md [hunk 1]
```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
 
 ## Evaluations
 
-Evaluations are a method for measuring the performance of your LLM application. For example, quality checks like failure to answer or topic relevancy are different types of evaluations that you can track for your LLM application.
+LLM Observability offers quality checks and out-of-the-box metrics to evaluate the quality and effectiveness of your LLM conversations, including assessments of sentiment, topic relevancy, and user satisfaction. With evaluations, you can understand the performance of conversations and enhance your LLM application's responses which improves the user experience and ensures valuable, accurate outputs.
```
Comment (neko-dd) [inline content/en/llm_observability/terms/_index.md:151]:
```suggestion
LLM Observability offers quality checks and out-of-the-box metrics to evaluate the quality and effectiveness of your LLM conversations, including assessments of sentiment, topic relevancy, and user satisfaction. With evaluations, you can understand the performance of conversations and enhance your LLM application's responses. This improves the user experience and ensures valuable, accurate outputs.
```

File: content/en/llm_observability/terms/_index.md [hunk 2]
```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
...
+
+By proactively scanning for sensitive data, LLM Observability ensures that conversations remain secure and compliant with data protection regulations. This additional layer of security reinforces Datadog's commitment to maintaining the confidentiality and integration of user interactions with LLMs.
+
+LLM Observability associates evaluations with individual spans so you can view the inputs and outputs that led to a specific evaluation. While Datadog provides out-of-the-box evaluations for your traces, you can also [submit your own evaluations][6] to LLM Observability.
+
+### Quality evaluations
+
+#### Topic Relevancy
+
+This check identifies and flags user inputs that deviate from the configured acceptable input topics, ensuring that interactions stay pertinent to the LLM's designated purpose and scope.
```
Comment (neko-dd) [inline content/en/llm_observability/terms/_index.md:165]:
```suggestion
This check identifies and flags user inputs that deviate from the configured acceptable input topics. This ensures that interactions stay pertinent to the LLM's designated purpose and scope.
```

File: content/en/llm_observability/terms/_index.md [hunk 3]
```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
...
+
+{{< img src="llm_observability/evaluations/toxicity.png" alt="A Toxicity evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}
+  
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input and Output | Evaluated using LLM | Toxicity flags any language or behavior that is harmful, offensive, or inappropriate, including but not limited to hate speech, harassment, threats, and other forms of harmful communication. |
+
+#### Prompt Injection
+
+This check identifies attempts to manipulate the LLM's responses or redirect the conversation in ways not intended by the original users. This check maintains the integrity and authenticity of interactions between users and the LLM.
```
Comment (neko-dd) [inline content/en/llm_observability/terms/_index.md:217]: Is this about a third party or is this about the original intent by the designer? I'm a bit confused.

File: content/en/llm_observability/terms/_index.md [hunk 4]
```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
...
+
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input | Evaluated using LLM | Prompt Injection Flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |
+
+#### Sensitive Data Scanning
+
+This check ensures that sensitive information is handled appropriately and securely, reducing the risk of data breaches or unauthorized access.
+
+{{< img src="llm_observability/evaluations/sensitive_data_scanning.png" alt="A Security & Safety evaluation detected by the Sensitive Data Scanner in LLM Observability" style="width:100%;" >}}
```
Comment (neko-dd) [inline content/en/llm_observability/terms/_index.md:229]:
```suggestion
{{< img src="llm_observability/evaluations/sensitive_data_scanning.png" alt="A Security and Safety evaluation detected by the Sensitive Data Scanner in LLM Observability" style="width:100%;" >}}
```

Comment (neko-dd) [reply content/en/llm_observability/terms/_index.md:229]: (unless this is a specific term we always use with an ampersand)

File: content/en/llm_observability/terms/_index.md [hunk 5]
```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
...
+
+#### Prompt Injection
+
+This check identifies attempts to manipulate the LLM's responses or redirect the conversation in ways not intended by the original users. This check maintains the integrity and authenticity of interactions between users and the LLM.
+
+{{< img src="llm_observability/evaluations/prompt_injection.png" alt="A Prompt Injection evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}
+
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input | Evaluated using LLM | Prompt Injection Flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:223]:
```suggestion
| Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |
```

File: content/en/llm_observability/terms/_index.md [hunk 6]
```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
...
+
+{{< img src="llm_observability/evaluations/toxicity.png" alt="A Toxicity evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}
+  
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input and Output | Evaluated using LLM | Toxicity flags any language or behavior that is harmful, offensive, or inappropriate, including but not limited to hate speech, harassment, threats, and other forms of harmful communication. |
+
+#### Prompt Injection
+
+This check identifies attempts by unauthorized or malicious users to manipulate the LLM's responses or redirect the conversation in ways not intended by the original user. This check maintains the integrity and authenticity of interactions between users and the LLM.
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:217]:
```suggestion
This check identifies attempts by unauthorized or malicious authors to manipulate the LLM's responses or redirect the conversation in ways not intended by the original author. This check maintains the integrity and authenticity of interactions between users and the LLM.
```

File: content/en/llm_observability/terms/_index.md [hunk 7]
```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
...
+
+{{< img src="llm_observability/evaluations/prompt_injection.png" alt="A Prompt Injection evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}
+
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:223]:
```suggestion
| Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or author. |
```

File: content/en/llm_observability/terms/_index.md [hunk 8]
```diff
@@ -148,11 +148,89 @@ You can instrument your LLM application to trace and group together all workflow
...
+
+#### Prompt Injection
+
+This check identifies attempts by unauthorized or malicious authors to manipulate the LLM's responses or redirect the conversation in ways not intended by the original author. This check maintains the integrity and authenticity of interactions between users and the LLM.
+
+{{< img src="llm_observability/evaluations/prompt_injection.png" alt="A Prompt Injection evaluation detected by an LLM in LLM Observability" style="width:100%;" >}}
+
+| Evaluation Stage | Evaluation Method | Evaluation Definition | 
+|---|---|---|
+| Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or author. |
```
Comment (alai97) [inline content/en/llm_observability/terms/_index.md:223]:
```suggestion
| Evaluated on Input | Evaluated using LLM | Prompt Injection flags any unauthorized or malicious insertion of prompts or cues into the conversation by an external party or user. |
```

**Pull request merged**